backend/src/backup/backup_bulk.rs

```diff
@@ -350,6 +350,11 @@ async fn perform_backup<Db: DbHandle>(
             .await;
 
         let mut tx = db.begin().await?;
+        let lock_package = crate::db::DatabaseModel::new()
```

Comment:
This should be the locking that we need for the system since we are needed to lock during the backup. Again this used to be covered by the system, and we don't want to remove the package while we are backing it up.